src/platform/site-wide/sass/modules/_m-veteran-banner.scss

```diff
@@ -0,0 +1,27 @@
+$veteran-banner-desktop-width: 1280px;
+$veteran-banner-desktop-height: 160px;
+$veteran-banner-mobile-width: 318px;
+$veteran-banner-mobile-height: 70px;
+
+.veteran-banner {
+  @include linear-gradient-background($white, $mm-gray);
+  margin-top: 2.5em;
+}
+
+.veteran-banner-image {
+  margin: 0 auto;
+  background-image: url(/img/homepage/veterans-banner-mobile.png);
+  background-repeat: no-repeat;
+  background-position: center bottom;
+  background-size: 100% auto;
+  height: $veteran-banner-mobile-height;
+  max-width: $veteran-banner-mobile-width;
+}
+
+@include media($small-screen) {
```

Comment:
Note that you could include this media query in the above class:

```scss
veteran-banner-image {
  margin: 0 auto;
  background-image: url(/img/homepage/veterans-banner-mobile.png);
  background-repeat: no-repeat;
  background-position: center bottom;
  background-size: 100% auto;
  height: $veteran-banner-mobile-height;
  max-width: $veteran-banner-mobile-width;
  @include media($small-screen) {
    background-image: url(/img/homepage/veterans-banner-desktop.png);
    height: $veteran-banner-desktop-height;
    max-width: $veteran-banner-desktop-width;
  }
}
```

I find that easier to read in general, since it keeps media queries nearby the styles they're modifying.

Reply:
Ah nice, yeah after converting to background-image I didn't think of that.